### PR TITLE
Adding a couple comments for ci review events

### PR DESCRIPTION
**Pull request by allanbolipata**
Added an `ETLGenericEvent` for just posting a message related to ETL jobs without message preformatting.

### CODE REVIEW

File: runner/tasks.py [hunk 1]
```diff
@@ -138,6 +138,8 @@ def process_triggers():
                 else:
                     operator_run.fail()
                     if job_group_id:
+                        e = OperatorRequestEvent(job_group_id, "CIReviewEvent: Operator Run %s failed" % str(operator_run.id)).to_dict()
```
Comment (allanbolipata) [inline runner/tasks.py:141]: Don't know how to test this

File: beagle_etl/tasks.py [hunk 2]
```diff
@@ -199,6 +199,8 @@ def _job_failed(self):
         if self.job.run == TYPES['REQUEST']:
             import time
             time.sleep(5)
+            e = ETLGenericEvent(self.job.job_group.id, "CIReviewEvent: ETL Job failed, likely child job import.").to_dict()
```
Comment (allanbolipata) [inline beagle_etl/tasks.py:202]: This is tested

File: beagle_etl/tasks.py [hunk 3]
```diff
@@ -199,6 +199,8 @@ def _job_failed(self):
         if self.job.run == TYPES['REQUEST']:
             import time
             time.sleep(5)
+            e = ETLGenericEvent(self.job.job_group.id, "[CIReviewEvent] ETL Job failed, likely child job import. Check pooled normal import, might already exist in database.").to_dict()
```
Comment (allanbolipata) [inline beagle_etl/tasks.py:202]: This has been tested

File: runner/tasks.py [hunk 4]
```diff
@@ -138,6 +138,8 @@ def process_triggers():
                 else:
                     operator_run.fail()
                     if job_group_id:
+                        e = OperatorRequestEvent(job_group_id, "[CIReviewEvent] Operator Run %s failed" % str(operator_run.id)).to_dict()
```
Comment (allanbolipata) [inline runner/tasks.py:141]: Not sure how to test this

File: beagle_etl/jobs/lims_etl_jobs.py [hunk 5]
```diff
@@ -88,19 +88,21 @@ def request_callback(request_id, job_group=None):
...
+        msg = "No operator defined for requestId %s with recipe %s" % (request_id, recipes[0])
+        logger.error(msg)
+        e = OperatorRequestEvent(job_group_id, "[CIReviewEvent] %s" % msg).to_dict()
         send_notification.delay(e)
         ci_review_e = SetCIReviewEvent(job_group_id).to_dict()
         send_notification.delay(ci_review_e)
-        raise FailedToSubmitToOperatorException("Not operator defined for recipe: %s" % recipes[0])
+        raise FailedToSubmitToOperatorException(msg)
     if not operator.active:
-        logger.info("Submitting request_id %s to %s operator" % (request_id, operator.class_name))
```
Comment (allanbolipata) [inline beagle_etl/jobs/lims_etl_jobs.py:98]: I moved this to `msg` and passing it around. I'm assuming the intent was to have the same string get logged by logger.info and also in `FailedToSubmitToOperatorException`; originally two different strings were reported to the logger and the exception.

File: notifier/event_handler/jira_event_handler/jira_event_handler.py [hunk 6]
```diff
@@ -41,6 +41,9 @@ def process_run_completed(self, event):
     def process_operator_request_event(self, event):
         self._add_comment_event(event)
 
+    def process_etl_job_failed_event(self, event):
```
Comment (sivkovic) [inline notifier/event_handler/jira_event_handler/jira_event_handler.py:44]: This method needs to be added to `notifier/event_handler/noop_event_handler/noop_event_handler.py` also.

Comment (allanbolipata) [reply notifier/event_handler/jira_event_handler/jira_event_handler.py:44]: done

**Pull request merged**